docs/core/install/linux-package-manager-centos7.md

```diff
@@ -3,14 +3,16 @@ title: Install .NET Core on CentOS 7 - package manager - .NET Core
 description: Use a package manager to install .NET Core SDK and runtime on CentOS 7.
 author: thraka
 ms.author: adegeo
-ms.date: 12/04/2019
+ms.date: 03/17/2020
 ---
 
 # CentOS 7 Package Manager - Install .NET Core
 
 [!INCLUDE [package-manager-switcher](./includes/package-manager-switcher.md)]
 
-This article describes how to use a package manager to install .NET Core on CentOS 7. If you're installing the runtime, we suggest you install the [ASP.NET Core runtime](#install-the-aspnet-core-runtime), as it includes both .NET Core and ASP.NET Core runtimes.
+This article describes how to use a package manager to install .NET Core on CentOS 7. Install the SDK (includes the runtime) if you want to make apps with .NET Core. Or, install the runtime if you only need to run apps. Install the SDK (includes the runtime) if you want to make apps with .NET Core. Or, install the runtime if you only need to run apps. If you're installing the runtime, we suggest you install the [ASP.NET Core runtime](#install-the-aspnet-core-runtime), as it includes both .NET Core and ASP.NET Core runtimes.
+
+If you've already installed the SDK or runtime, use the `dotnet --list-sdks` and `dotnet --list-runtimes` commands to see which versions are installed. For more information, see [How to check that .NET Core is already installed](how-to-detect-installed-versions.md?pivots=os-linux).
```

Comment:
Since this paragraph is repeated 16 times, it's probably better to make it an include file.